proposals/Level-1.md

```diff
@@ -51,13 +51,12 @@ exceptions.  For embedders that have garbage collection (such as JavaScript),
 this is not a problem.
 
 However, not all embedders may have a garbage collector. For this reason,
-WebAssembly exceptions are designed to allow the use of reference counters (to
-exceptions) to perform the the garbage collection in the embedder.
+WebAssembly exceptions are designed to allow the use of reference counters to
+perform the the garbage collection in the embedder.
 
-To do this, WebAssembly exceptions are immutable once created. This avoids
-cyclic data structures that can't be garbage collected. It also means that
-exceptions can't be stored into WebAssembly's linear memory. The rationale for
-this is twofold:
+To do this, WebAssembly exceptions are immutable once created, to avoid cyclic
+data structures that can't be garbage collected. It also means that exceptions
```

Comment:
nit: for spec text, I think spelling out contractions is better. i.e. "can't" -> "cannot"